src/xunit.performance.api/Table/DataTable.cs

```diff
@@ -38,21 +37,24 @@ public IEnumerable<Row> Rows
             get { return _Rows; }
         }
 
-        public void WriteToCSV(string fullFilePath)
+        public void WriteToCSV(string fullFilePath, bool sort = true)
```

Comment:
Agree. This was the fastest way to give corefxlab the feature.
I still want to refactor all the output story for the API, so I would like to do this in the near future.